GlobalViolence/R/DataPrep_Graduate.R

```diff
@@ -14,7 +14,13 @@ library(DemoTools)
 library(MortalitySmooth)
 library(reshape2)
 library(magrittr)
+
 # TR: any other cases?
+# VDL: I don´t think we should this. We actually need both packages to run 
+# library(parallel) this because we can use your mc.cores = (detectCores() - 1) command
+# library(parallelsugar) this because of Windowns operational system
```

Comment:
I see. So we should always do `library(parallel)` first, and then add in 
`if(.Platform$OS.type == "windows"){
  library(parallelsugar)
}`